src/components/tag/_tag.scss

```diff
@@ -86,10 +86,11 @@
     &:not(:first-child) {
       margin-left: 0;
     }
-  }
 
-  .#{$prefix}--tag--basic {
-    @include tag-theme--x($ibm-colors__gray-20, $ibm-colors__gray-100);
+    &--basic,
+    &.#{$prefix}--skeleton {
+      @include tag-theme--x($bg-color: $ui-03, $text-color: $text-01);
```

Comment:
Is this not using theme-white? @joshblack I thought for theme-white the `$ui-03` hex code was `#dcdcdc`, and `$text-01` is `#171717`. It seems that it should be using `$hover-ui` which corresponds to `#e5e5e5`, and either keep `$ibm-colors__gray-30` or use a token that corresponds to `#bebebe`. Unless I'm missing something, I could be totally wrong.

Reply:
@aagonzales @laurenmrice Any thoughts here...? Thanks!

Reply:
@jnm2377 I'm happy to update with the correct values - The reasoning here was to use a token role that was closest to the color for multiple themes ('Gray 100' specifically) while preventing the use a token role where the semantics didn't make sense.

For example `$active-ui` and `$hover-ui` didn't make sense to me here because these did not correspond to an active or hover state for a base state.

Reply:
@SimonFinney that makes sense. I agree that `$active-ui` and `$hover-ui` don't make sense as semantic tokens here. I just wasn't sure if that was an intentional change or not. Thanks for the explanation.

Reply:
We could add one more new token that's like `$skeleton-ui`